src/MagnumExternal/TinyGltf/json.hpp

```diff
@@ -1,7 +1,7 @@
 /*
     __ _____ _____ _____
  __|  |   __|     |   | |  JSON for Modern C++
-|  |  |__   |  |  | | | |  version 3.3.0
+|  |  |__   |  |  | | | |  version 3.5.0
```

Comment:
We need less of this thing, not more 😅. Any comment about why is it needed?

Reminds me of https://twitter.com/CaliaDomenico/status/1207922171902906369

Reply:
I just updated it to the latest version that tiny_gltf uses (the exact file bundled with tiny_gltf). I can revert this, though